frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx

```diff
@@ -110,6 +110,7 @@ export const ParametersWidgetContainer = styled(FullWidthContainer)`
   ${({ isSticky, isNavbarOpen }) =>
     isSticky &&
     isNavbarOpen &&
+    !IFRAMED &&
```

Comment:
Without this, dashboard filters in embedded mode look incorrect
<img width="1362" alt="Screenshot 2022-04-23 at 03 38 07" src="https://user-images.githubusercontent.com/14301985/164832727-0df22387-fe02-45dc-a322-eacb75fa03da.png">